lib/editor/components/timetable/EditableCell.js

```diff
@@ -338,8 +320,7 @@ export default class EditableCell extends Component {
         type='text' />
       : <div
         className='cell-div noselect'
-        style={divStyle}
-        >
+        style={divStyle}>
```

Comment:
We should use a react/jsx linter because I prefer the closing bracket being tag-aligned. 
 The default [linting rule on bracket location](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-closing-bracket-location.md) seems to contradict both of our preferences.  :)